en/docs/administer/changing-datasource-bpsds.md

```diff
@@ -0,0 +1,106 @@
+# Changing the Default Datasource for BPS.
+
+By default WSO2 Identity Server use Embedded H2 database as the BPS datasource. However you
+can change this to any database type that is supported by WSO2 Identity Server.
+
+-  Following are the sample configuration for each database type.
```

Comment:
Better to have them in sections, instead of tabs.

Reply:
done